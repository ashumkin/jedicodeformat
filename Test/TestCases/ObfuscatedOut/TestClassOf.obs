unit testclassof;interface uses classes;type mystringlistclass=class of tstringlist
;mystringnamespacedlistclass=class of classes.tstringlist;implementation end. 