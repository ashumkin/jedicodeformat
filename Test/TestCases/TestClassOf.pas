unit TestClassOf;

{ This unit compiles but is not semantically meaningfull
 it is test cases for the code formatting utility
}

interface

uses classes;

type
  MyStringListClass = class of TStringList;
  MyStringNamespacedListClass = class of Classes.TStringList;

implementation

end.
