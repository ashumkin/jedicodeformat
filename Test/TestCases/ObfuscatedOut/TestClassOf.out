unit testclassof;

interface

uses Classes;

type
  mystringlistclass = class of TStringList;
  mystringnamespacedlistclass = class of Classes.TStringList;

implementation

end.
 